src/hooks/useChangePwdGuide.ts

```diff
@@ -0,0 +1,92 @@
+import { computed, onMounted, Ref, watch, ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+import { ElMessageBox } from 'element-plus'
+import { useRouter, useRoute } from 'vue-router'
+import { User } from '@/types/systemModule'
+import useI18nTl from './useI18nTl'
+
+export default (): void => {
+  const store = useStore()
+  const router = useRouter()
+  const { tl } = useI18nTl('Base')
+
+  const isUsingDefaultPwd = computed(() => {
+    return store.state.user.isUsingDefaultPwd
+  })
+
+  const popupMessageBox = () => {
+    ElMessageBox.alert(tl('defaultPwdTip'), {
```

Comment:
type: info?

Reply:
There is no info type, although it is an alert type but there is no red UI